src/contents/all/components/knot/Knot/Knot.scss

```diff
@@ -45,3 +45,56 @@
     justify-content: flex-end;
   }
 }
+
```

Comment:
stealing this from mainframe to get delete knot popup... definitely not implementing this correctly but i imagine i'll need it at some point if we want to mimic mainframe UX?

Reply:
Yea think since we don't have reply or liking, we just need a simple delete button that confirms. So we'll definitely want to leverage [ant design Popconfirm](https://ant.design/components/popconfirm/). We can use 4.x too since this library is leveraging Ant design 4

Reply:
popconfirm API is soooo sexy wtf